xacc/compiler/qalloc.hpp

```diff
@@ -14,6 +14,7 @@
 #define XACC_QALLOC_HPP_
 
 #include <map>
+#include <string>
```

Comment:
I guess that gcc10 changed some internal dependency b/w standard headers.
For some reason, my Mac updated gcc without my knowledge :) hence seeing this. Worth fixing.